frontend/components/Personcard.js

```diff
@@ -1,22 +1,14 @@
-import style from '../styles/Personcard.module.css'
-
 const PersonCard = ({ person }) => {
   return (
-    <article className={style.card}>
-      <div className={style.container}>
-        <img
-          src={person.photoImage.url}
-          width='240'
-          height='240'
-          preserveAspectRatio='xMidYMid slice'
-        />
-      </div>
-      <div className={style.bottom}>
-        <h5><strong>{person.FirstName} {person.LastName}</strong></h5>
-        <hr className={style.horizontalbar} />
-        <h6 className='beaverorange'>{person.personClass.Name}</h6>
+    <div className='col'>
+      <div className='card'>
+        <img src={person.photoImage.url} className='card-img-top' alt='Team member portrait' />
+        <div className='card-body'>
+          <h5 className='card-title'>{person.formattedPersonName}</h5>
+          <h6 className='card-subtitle mb-2 beaverorange'>{person.personClass.Name}</h6>
```

Comment:
This is not a blocking error, but as a future item, we need to look into the usage of header tags (h1-h6) and decide if they are necessary in that context.

Reply:
Good catch, for now I'll edit the cards to follow the same formatting as the project cards. The names will be h5, and the description/sub-text will be paragraph text with the 'text-muted' subclass.